docs/modules/ROOT/pages/how-to-guides/proc_upgrade_build_pipeline.adoc

```diff
@@ -51,3 +51,20 @@ After upgrading the build pipeline, you can also customize it:
 {ProductName} creates pull requests that contain updates to the pipeline definitions under the `.tekton` directory.
 Run the tests on these pull requests by creating a comment containing `/ok-to-test` and then merging the changes when the tests pass.
 ====
+
+.Security
+To reinforce the security of your custom build pipeline, complete the following steps:
```

Comment:
Could we add "further" here? "To further reinforce the security of your custom build pipeline..." Because the earlier part of the doc states that you customize in the first place for stronger security. So I think we should clarify that this section tells readers how to reinforce their security even more :)